README.md

```diff
@@ -14,9 +18,103 @@ Communicate information about background jobs.
 
 1. Run `bundle install` from your shell.
 
+1. Generate migrations so `FrenchToast` can render information when javascript isn't available.
+
+```
+rails generate french_toast:install
+rake db:migrate
+```
+
 ## Usage
 
-TODO
+### General French Toast
+
+Simplest thing you can do is use the default french toast helper in a view
+
+```erb
+# application.html.erb
+<%= french_toast %>
+```
+
+Create a partial in `views/french_toast`
+
+```erb
+# _foo.html.erb
+<div>You'll see me eventually! <%= your_name %></div>
+```
+
+Finally, in a background job call the `notify` method with the template you'd like to use, with an optional payload
+
+```ruby
+#... something_job.rb
+def perform
+  FrenchToast.notify("foo", payload: { your_name: "Jim-bob" })
+end
+```
+
+```javascript
+document.addEventListener('french_toast:notify',
+  function(json) {
+    alert("Your foo is done, " + json["your_name"]);
+  });
+```
```

Comment:
You know what this is. This was something that Mike and I wrote out as a possible interaction with a custom event listener, if there were additional things you wanted to do.